--- /mnt/data/epgwork/epgstation/source/client-src/views/WatchOnAir.vue	2022-01-10 10:08:39.000000000 +0000
+++ /mnt/data/epgstation-jikkyo-live-patch/client/src/views/WatchOnAir.vue	2026-08-31 10:48:27.528023515 +0000
@@ -3,7 +3,11 @@
         <TitleBar title="視聴"></TitleBar>
         <transition name="page">
             <div class="video-container-wrap mx-auto">
-                <VideoContainer v-if="videoParam !== null" v-bind:videoParam="videoParam"></VideoContainer>
+                <VideoContainer
+                    v-if="videoParam !== null"
+                    v-bind:videoParam="videoParam"
+                    v-bind:jikkyoChannelId="watchParam === null ? undefined : watchParam.channel"
+                ></VideoContainer>
                 <WatchOnAirInfoCard v-if="watchParam !== null" v-bind:channel="watchParam.channel" v-bind:mode="watchParam.mode"></WatchOnAirInfoCard>
                 <div style="visibility: hidden">dummy</div>
             </div>
